web/src/routes/(user)/utilities/duplicates/[[photos=photos]]/[[assetId=id]]/+page.svelte

```diff
@@ -132,6 +164,12 @@
         {$t('keep_all')}
       </div>
     </LinkButton>
+    <LinkButton on:click={() => handleStackAll()} disabled={!hasDuplicates}>
```

Comment:
I think it should be disabled if more than one asset is selected and should make the selected asset the primary asset of the stack.